nginx.conf.template

```diff
@@ -162,8 +169,8 @@ http {
 
     location @proxy_to_app {
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-      # enable this if and only if you use HTTPS
-      # proxy_set_header X-Forwarded-Proto https;
+      # enable the following if and only if you use HTTPS
```

Comment:
Only the next 1 line dealing with HTTPS, I kept it in case we want to test without HTTPS.